service/application/src/main/java/com/solace/maas/ep/event/management/agent/service/ScanService.java

```diff
@@ -244,16 +244,6 @@ public Optional<ScanEntity> findById(String scanId) {
         return repository.findById(scanId);
     }
 
-    protected Exchange scan(String groupId, String scanId, RouteEntity route,
```

Comment:
This old scan method has been removed since it is not used by any other methods/classes